Dockerfile

```diff
@@ -4,7 +4,8 @@ SHELL ["/bin/bash", "-c"]
 
 ENV MLSERVER_MODELS_DIR=/mnt/models \
     MLSERVER_ENV_TARBALL=/mnt/models/environment.tar.gz \
-    PATH=/home/default/.local/bin:$PATH
+    PATH=/home/default/.local/bin:$PATH \
+    MLSERVER_VERSION=0.4.0
```

Comment:
not sure if there is a better way of defining the version to be used. i.e. to fetch it from where the version is actually defined?

Reply:
Good point. I'd install the version that it's copied locally to the Docker image rather than fetching one from PyPi. That way we ensure that the Docker image reflects whatever is on the repo.